core/jvm/src/main/scala/cats/effect/unsafe/IORuntimeBuilderPlatform.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020-2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect.unsafe
+import scala.concurrent.ExecutionContext
+
+private[unsafe] abstract class IORuntimeBuilderPlatform {
```

Comment:
```suggestion
private[unsafe] abstract class IORuntimeBuilderPlatform { self: IORuntimeBuilder =>
```

Then, you will be able to access all of its member variables, without need to request them as parameters in the `build` method below.

Reply:
For some reason, I thought it wouldn't work when I first tried it. Seems like my IDE was playing tricks on me. Done now :+1: